test/Sentry.NLog.Tests/SentryTargetTests.cs

```diff
@@ -103,7 +103,9 @@ public void Can_configure_from_xml_file()
                     </targets>
                 </nlog>";
 
-            var c = XmlLoggingConfiguration.CreateFromXmlString(configXml);
+            var stringReader = new System.IO.StringReader(configXml);
```

Comment:
What's the purpose of this change?

Reply:
I'm downgrading the NLog-version from NLog 4.6 to 4.5.11. This helper-method is not available in 4.5.11

Reply:
LGTM